app/controllers/api/v2/split_registries_controller.rb

```diff
@@ -2,6 +2,6 @@ class Api::V2::SplitRegistriesController < UnauthenticatedApiController
   include CorsSupport
 
   def show
-    @split_registry = SplitRegistry.instance
+    @split_registry = SplitRegistry.new(as_of: Time.zone.now)
```

Comment:
Hey so given the fact that if I merge and deploy this, the behavior will be on my head I looked a little deeper at the current behavior here. It's my understanding that the closest to zero behavior change for this call would be to pass `as_of: nil`, because `instance` used to mean [`Split.for_presentation()`](https://github.com/Betterment/test_track/pull/133/files#diff-61ce81f8fc268c7cd2c4f9ef065c7755L5), and `Split.for_presentation()` would call [Split.active()](https://github.com/Betterment/test_track/blob/master/app/models/split.rb#L34) which would apply the condition `where(finished_at: nil)`. As it is this would end up applying the condition `where('splits.finished_at is null or splits.finished_at > ?', Time.zone.now)`, which is sort of spurious since we'd expect there to be no splits finished in the future, but clock error and all that jazz could make that a thing under some circumstances.

Granted this would fail validation for `SplitRegistry`, if we actually `valid?`, which we're not doing here. Do we care? I guess not. Ok I'm on board with this. Basically this is behaviorally correct under all normal circumstances and hard to behaviorally regress, and the whole v2 api is soft deprecated at this point anyway so the blast radius of a regression here is low. Ok. I'mma merge this.